deploy/charts/cert-manager/values.yaml

```diff
@@ -17,6 +17,7 @@ global:
   podSecurityPolicy:
     enabled: false
 
+  # Set the verbosity of cert-manager. Range of 0 - 6 with 6 being the most verbose.
```

Comment:
It’s technically possible to keep going. There’s no ceiling to v - I think if you set v=8 you start getting hex dumps of every HTTP request the k8s apimachinery makes

Reply:
Indeed, but 6 is the highest that cert-manager uses itself at least. Happy to make a note that it's basically limitless if you want?